### PR TITLE
fix(walletconnect): copy address issue

### PR DESCRIPTION
**Pull request by Abhishekkochar**
#### What this PR does / why we need it:
This fixes Issue #121. Users will be able to copy the clipboard if connecting via wallet connect. 
#### Which issue(s) does this PR fixes?:
Issue #121 

Fixes #

### CODE REVIEW

**Review by gjj — DISMISSED** [review]

File: .env
Comment (Abhishekkochar) [reply .env]: Yes, you are correct. It has been removed

**Review by chee-chyuan — APPROVED** [review]

File: package.json
Comment (Abhishekkochar) [reply package.json:71]: Fixed

**Comment by Abhishekkochar** [comment]
@gjj Pls have a quick look. I will merge once approved. Thannks

**Review by gjj — APPROVED** [review]

**Pull request merged**
